### ISSUE TITLE
White-listed mac addresses case sensitive?

### ISSUE THREAD

**Issue by GrumpyMeow** [shmuelzon/esp32-ble2mqtt#55]
Can it be that the the mac addresses are case-sensitive and need to be lowercased?

**Comment by shmuelzon** [comment]
It can, and [is](https://github.com/shmuelzon/esp32-ble2mqtt/blob/8418e34e7792d19199f0c74b1c8daa00be3ee780/main/config.c#L126)...
Internally, all MAC addresses are lower case, the configuration should be the same. It can be easily modified to be case insensitive, but is that really needed?

**Comment by shmuelzon** [comment]
Done

**Comment by GrumpyMeow** [comment]
You’re right that it’s not really needed. But I did run into this issue when I copied the MAC address from an application. This change will increase the user friendliness.
